packages/cli/commands/project/download.js

```diff
@@ -22,18 +22,24 @@ const {
   ensureProjectExists,
 } = require('../../lib/projects');
 const { loadAndValidateOptions } = require('../../lib/validation');
+const {
+  downloadProjectPrompt,
+} = require('../../lib/prompts/downloadProjectPrompt');
 const { i18n } = require('@hubspot/cli-lib/lib/lang');
 
 const i18nKey = 'cli.commands.project.subcommands.download';
 const { EXIT_CODES } = require('../../lib/enums/exitCodes');
 
-exports.command = 'download <name> [dest]';
+exports.command = 'download';
```

Comment:
This can be `download [--project]`. That's the format that we've been using to highlight command options that are semi-required. Semi-required options are options that can be omitted, but omitting them will result in a prompt.

Here's an example in [hs project deploy](https://github.com/HubSpot/hubspot-cli/blob/master/packages/cli/commands/project/deploy.js#L23). Also, as part of this pattern we're including the option in the prompt so that the user can quickly learn how to skip the prompt in the next run ([example](https://github.com/HubSpot/hubspot-cli/blob/master/packages/cli-lib/lang/en.lyaml#L1036)). I did a quick look and it seems like we're using `project` as the option in our other project commands.